src/html/HTMLWriter.jl

```diff
@@ -1790,6 +1795,27 @@ function write_html(ctx::HTMLContext, navnode::Documenter.NavNode, page_html::DO
     return true
 end
 
+"""
+Calculates and converts bytes to appropriate format.
+"""
+function format_units(size, unit = "bytes")
```

Comment:
Just make this function return a `String`. It doesn't get used elsewhere

Reply:
Since this is a utility function, I would argue against doing this, as it can be useful somewhere else in the future.

Reply:
I'm with @odow here: it's better to have the string formatting complexity just once in this utility function, rather than reproduced three times in `size_threshold_msg`. The latter is not very readable right now. The function is quite simple and internal, so we can always change things up if we need to reuse this somehow.

Reply:
> The function is quite simple and internal, so we can always change things up if we need to reuse this somehow.

Alright, no problem will refactor and update the PR.